docs/developer-docs/backend/motoko/define-an-actor.mdx

```diff
@@ -1,8 +1,12 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
 
 # 14: Querying using an actor
 
+<MarkdownChipRow labels={["Intermediate", "Motoko", "Tutorial"]} />
```

Comment:
for SEO, I think these same tags should be added into the frontmatter as keywords.